save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/vuln/VulnerabilityGeneralInfoProps.kt

```diff
@@ -221,14 +221,13 @@ val vulnerabilityGeneralInfoProps: FC<VulnerabilityGeneralInfoProps> = FC { prop
                         onChange = { event ->
                             props.setVulnerability { vulnerability ->
                                 vulnerability?.copy(
-                                        cosv = cosv.copy(aliases = event.target.value.split(PRETTY_DELIMITER))
+                                    cosv = cosv.copy(aliases = event.target.value.split(PRETTY_DELIMITER))
```

Comment:
maybe add checking on duplicates in aliases?

Reply:
add `distinct`